client/my-sites/plans-single-products/product.jsx

```diff
@@ -3,22 +3,119 @@
  */
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import { localize } from 'i18n-calypso';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import PlanPrice from 'my-sites/plan-price';
+import ProductOption from './product-option';
 
 class PlansSingleProduct extends Component {
 	static propTypes = {
-		heading: PropTypes.string.isRequired,
+		billingTimeFrame: PropTypes.string,
+		currencyCode: PropTypes.string,
+		discountedPrice: PropTypes.number,
+		fullPrice: PropTypes.number,
+		isPlaceholder: PropTypes.bool,
+		moreInfoLabel: PropTypes.string,
+		onSelect: PropTypes.func,
+		options: PropTypes.array,
+		optionsHeading: PropTypes.string,
+		productDescription: PropTypes.string,
+		slug: PropTypes.string,
+		title: PropTypes.string,
+	};
+
+	static defaultProps = {
+		isPlaceholder: false,
 	};
 
+	renderPriceGroup() {
```

Comment:
I see some overlap with the existing `<PlanFeatures />` component. Could we have borrowed something from there instead of inventing our own components for so many things?

Reply:
@delawski lets see if we can use similar props here. Does that even make sense?

Reply:
FWIW, it's not only about the props, I see some similarity between the components, so I think we could reuse more than just the props.